.github/workflows/build_feature_server.yml

```diff
@@ -0,0 +1,25 @@
+name: build_feature_server_aws
+
+on: [push, pull_request]
+
+jobs:
+  build-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          clean: false
```

Comment:
why do we have clean: false here but not in other workflows?

Reply:
It isn't necessary, I was experimenting locally and forgot to remove it. Will remove it for final version.